### ISSUE TITLE
USE_TABLE_INDEX=True when swapping a space during a reboot causes a lock issue. 

### ISSUE THREAD

**Issue by nsthorat** [lilacai/lilac#1078]
This is because two instances are pointing to the same database instance and we ask for a file pointer in write mode.

Both the old instance and the new instance crash. Restarting solves the issue.

**Comment by dsmilkov** [comment]
Was this fixed?

**Comment by brilee** [comment]
Yes, we fixed it by not having the hf_docker_start script attempt to touch any databases during the startup.

That being said, it's still possible since we have no guarantees on how quickly HF will kill the previous instance - it may still exist and hold onto file locks, but just not be connected to the broader internet.
